Makefile

```diff
@@ -175,6 +175,8 @@ run_tests.acceptance: ## Runs the acceptance tests for the parsed suite
 	else \
 		export PATH=$(shell cd .. && pwd)/oppia_tools/node-16.13.0/bin:$(PATH); \
 	fi
+# Adding env variable for mobile view
+	@export MOBILE=${MOBILE:-false}
```

Comment:
Please document this flag in the comments under `run_tests.acceptance` above

Reply:
Done.